src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ResendToGapsAboutToSubmitHandler.java

```diff
@@ -64,7 +65,8 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                 if (gapsSwitchOverFeatureEnabled && sscsCaseData.getSchedulingAndListingFields().getHearingRoute() == LIST_ASSIST) {
                     log.info("HearingRoute is ListAssist for CaseID: {}. Sending ListAssist cancellation message.", caseDetails.getId());
                     sscsCaseData.getSchedulingAndListingFields().setHearingRoute(HearingRoute.GAPS);
-                    hearingMessageHelper.sendListAssistCancelHearingMessage(sscsCaseData.getCcdCaseId());
+                    hearingMessageHelper.sendListAssistCancelHearingMessage(sscsCaseData.getCcdCaseId(),
+                            CancellationReason.OTHER);
```

Comment:
Not something to do with this PR but the one you have extended. In this handler we did not send the cancellation reason as Other. The cancellation reason field was introduced later in SSCS-10335. Please can you make it null instead of OTHER.

Reply:
Updated to null - Thank you!